packages/pocket-ic/BUILD.bazel

```diff
@@ -95,6 +97,8 @@ test_using_pocket_ic_server(
     srcs = ["tests/restart.rs"],
     # Override the default ["manual"] extra_HEAD_tags to ensure this test runs on CI.
     extra_HEAD_tags = [],
+    # Override the default [] extra_mainnet_tags to skip this test on CI.
+    extra_mainnet_tags = ["manual"],
```

Comment:
Could we please do the same in `//rs/pocket_ic_server` since running those tests against the mainnet server has the same downside as here?

Reply:
Done in https://github.com/dfinity/ic/pull/1468/commits/e740d31246d9a2edf0161038e9e0465e01a8fe53.

Reply:
Thank you!